File: QTD.SPA/src/app/_DtoModels/ClientsSettingsNotification/ClientSettings_Notification_CustomSetting.ts
export class ClientSettings_Notification_CustomSetting{
    clientSettingsNotificationId: number;
    key: string;
    value: string;

    constructor (key, value, clientSettingsNotificationId)
    {
      this.key = key;
      this.value = value;
      this.clientSettingsNotificationId = clientSettingsNotificationId;
    }
}

File: QTD2.Domain/Entities/Core/ClientSettings_Notification/ClientSettings_Notification_CustomSetting.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Domain.Entities.Core
{
    public class ClientSettings_Notification_CustomSetting : Common.Entity
    {
        public int ClientSettingsNotificationId { get; set; }
        public string Key { get; set; }
        public string Value { get; set; }

        public ClientSettings_Notification_CustomSetting() { }

        public ClientSettings_Notification_CustomSetting(string key, string value)
        {
            Key = key;
            Value = value;
        }

        public void UpdateValue(string value, string modifiedBy)
        {
            Value = value;
            ModifiedBy = modifiedBy;
            ModifiedDate = DateTime.Now;
        }
    }
}
